### PR TITLE
refactor: remove the unneeded aliases for types

### PR DESCRIPTION
**Pull request by yutianwu**
### Description

This pr will remove the unneeded aliases.

### Changes

Notable changes:
*  remove the unneeded aliases for types

### CODE REVIEW

File: common/chain/pubdata_helper.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 	"github.com/bnb-chain/zkbnb/types"
 )
 
-func ParseRegisterZnsPubData(pubData []byte) (tx *types.RegisterZnsTxInfo, err error) {
+func ParseRegisterZnsPubData(pubData []byte) (tx *legendTxTypes.RegisterZnsTxInfo, err error) {
```
Comment (forcodedancing) [inline common/chain/pubdata_helper.go:31]: Do we need to rename `legendTxTypes` to other name? `legend` word is wired now.

File: common/prove/deposit.go [hunk 2]
```diff
@@ -37,8 +38,8 @@ func (w *WitnessHelper) constructDepositTxWitness(cryptoTx *TxWitness, oTx *Tx)
 	return cryptoTx, nil
 }
 
-func toCryptoDepositTx(txInfo *legendTxTypes.DepositTxInfo) (info *CryptoDepositTx, err error) {
-	info = &CryptoDepositTx{
+func toCryptoDepositTx(txInfo *legendTxTypes.DepositTxInfo) (info *std.DepositTx, err error) {
+	info = &std.DepositTx{
```
Comment (forcodedancing) [inline common/prove/deposit.go:42]: Question: why the package `github.com/bnb-chain/zkbnb-crypto/legend/circuit/bn254/std` is named as `std`?

File: common/chain/pubdata_helper.go (hunk 1 above)
Comment (guagualvcha) [reply common/chain/pubdata_helper.go:31]: Agree

Comment (guagualvcha) [inline common/chain/pubdata_helper.go:31]: Can you help restructure the packages layout of `zkbnb-crypto` before move on this PR? `Legend` seems meanless in our solution.

**Review by guagualvcha — APPROVED** [review]

**Review by forcodedancing — APPROVED** [review]

**Pull request merged**